src/universe/analytics.ts

```diff
@@ -99,6 +110,9 @@ export function analytics (this: Universe): UniverseAnalytics {
     },
     subscriberMetrics: async (options?: UniverseAnalyticsOptions): Promise<SubscriberMetrics> => {
       return await makeAnalyticsRequest<SubscriberMetrics, undefined>('/universe/subscriptions')
+    },
+    revenueMetrics: async (options: UniverseRevenueMetricsOptions): Promise<RevenueMetrics> => {
+      return await makeAnalyticsRequest<RevenueMetrics, undefined>(`/attribution/${options?.version}/${options.start.toISOString()}/${options.end.toISOString()}`)
```

Comment:
todo: We are missing the 2nd argument here to pass the options, and the 2nd type then can be `UniverseRevenueMetricsOptions`